.travis.yml

```diff
@@ -32,18 +33,20 @@ matrix:
 
 before_install:
   - travis_retry composer self-update
-  - if [[ $TRAVIS_PHP_VERSION != "hhvm" && $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini ; fi
+  - phpenv config-rm xdebug.ini || true
 
 install:
   - if [[ $DEPS == 'latest' ]]; then travis_retry composer update --no-interaction ; fi
   - if [[ $DEPS == 'lowest' ]]; then travis_retry composer update --prefer-lowest --prefer-stable --no-interaction ; fi
   - if [[ $DEPS == 'locked' ]]; then travis_retry composer install --no-interaction --ignore-platform-reqs ; fi
-  - composer show
+  - stty cols 120
+  - COLUMNS=120 composer show
```

Comment:
Awesome thanks, been running into this issue but haven't found a solution yet. Thank you for coming up with one :+1: